modules/installation-disk-partitioning-upi-templates.adoc

```diff
@@ -35,6 +35,54 @@ Because `/var` must be in place before a fresh installation of {op-system-first}
 If you follow the steps to create a separate `/var` partition in this procedure, it is not necessary to create the Kubernetes manifest and Ignition config files again as described later in this section.
 ====
 
+.Prerequisites
+
+* If container storage is on the root partition:
+** Then, ensure that this root partition is mounted with the "pquota" option by including the `rootflags=pquota` option in the grub command line.
+
+* If the container storage is on a partition mounted by "/etc/fstab":
```

Comment:
s/partition mounted by/partition that is mounted by

s/"/etc/fstab"/`/etc/fstab`